.github/workflows/coverage.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Install cargo-llvm-cov
         run: curl -LsSf https://github.com/taiki-e/cargo-llvm-cov/releases/latest/download/cargo-llvm-cov-x86_64-unknown-linux-gnu.tar.gz | tar xzf - -C ~/.cargo/bin
       - name: Generate code coverage
-        run: cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
+        run: RUST_LOG=lib_didethresolver=trace cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
```

Comment:
I'm a bit concerned that trace will be too verbose for CI.  Github will limit the amount of data in the ci and this will also slow down the ci.    Is there any output in particular you would like to capture through this?

Reply:
it's mostly so that codecov doesn't flag `log::` as untested, can remove this or lower the verbosity if needed

Reply:
Is it possible to configure codecov to ignore `log::` instead?

Reply:
this is a good suggestion, after a bit of searching it does not seem like ignoring lines like this is supported. Ignoring functions is supported only in nightly, but even that wouldn't cover this case https://github.com/taiki-e/cargo-llvm-cov

Reply:
I'll just lower the verbosity for now, maybe revisit later